.eslintrc.js

```diff
@@ -1,22 +1,89 @@
 // eslint-disable-next-line no-undef
 module.exports = {
-  env: {
-    browser: true,
-    es2021: true,
-  },
-  extends: [
-    // "airbnb", // TODO: activate this styleset
-    "eslint:recommended",
-    "plugin:vue/strongly-recommended",
-    // "plugin:prettier/recommended", // TODO: consider activating
-  ],
-  parserOptions: {
-    ecmaVersion: "latest",
-    sourceType: "module",
-  },
-  plugins: ["vue"],
-  rules: {},
-  globals: {
-    $nuxt: true,
-  },
+
+    env: {
+
+        browser: true,
+        es2021: true,
+    },
+
+    extends: [
+
+        // "airbnb", // TODO: activate this styleset
+        "eslint:recommended",
+        "plugin:vue/strongly-recommended",
+        // "plugin:prettier/recommended", // TODO: consider activating
+    ],
+
+    // Known globals
+    globals: {
+
+        $nuxt: true,
+
+        // 'structuredClone' is available as of ES6 (ca. 2015), but ESLint does not yet know about it
+        structuredClone: true
+    },    
+
+    parserOptions: {
+
+        ecmaVersion: "latest",
+        sourceType: "module",
+    },
+
+    plugins: ["vue"],
+
+    rules: {
+
+        "no-unused-vars": [ "error", {
+
+            // Do not check for unused variables in argument lists
+            args: "none"
+        }],
+
+        "vue/html-self-closing": [ "error", {
+
+            html: {
+
+                // Don't enforce self closing on custom components that contain no child tags 
+                component: "any",
+
+                // Enforce self-closing on void html elements (i.e. 'input')
+                void: "always"
+            }
+        }],
+
+        "vue/html-closing-bracket-newline": [ "warn", {
+
+            // Disallow line breaks before the closing bracket on a multiline opening tag
+            multiline: "never"
+        }],        
+
+        // Four-space indentation
+        "vue/html-indent": [ "warn", 4, {
+            
+            // Attributes do not have to be aligned vertically
+            alignAttributesVertically: false
+        }],
+
+        "vue/max-attributes-per-line": [ "error", {
+
+            // Maximum number of attributes on a single line opening tag
+            "singleline": {
+                "max": 4
+            }
+        }],        
+
+        "vue/multiline-html-element-content-newline": [ "warn", {
+
+            // Allows blank lines in multiline tags
+            allowEmptyLines: true
+        }],
+
+        "vue/no-unused-vars": [ "error", {
+
+            // Ignore unused variables beginning with '_' character
+            // (Currently just used for unused variables in v-for statements)
+            "ignorePattern": "^_"
+        }]
+    }
```

Comment:
@surchs I've changed these to warnings and added some specific rules about indent, vue script-indent, comma spacing, and trailing commas.

The idea here is to explicit about the rules we want to enforce even though they might be default in eslint and vue-eslint rules?

Reply:
Cool. No, my comment was about the fact that when I run eslint with this config locally, I don't get yelled at for trailing whitespace and I can chose what level of indentation I want for each line as I see fit. I think we should pick some indentation style and remove trailing whitespaces so that its consistent in the code and we don't get diffs because one of us accidentally added or removed some whitespace.